### PR TITLE
Addition of KY state/county scraper.

### PR DESCRIPTION
**Pull request by DatJord**
This modification adds a scraper for all counties in KY based on data from the Kentucky Cabinet for Health and Family Services. This data is updated from KCHFS each day at 5pm. Please review and let me know if there is anything else we need to add on this (the changes ended up being minimal).

Note there will very likely be better sources to come along down the road for this data. But this should give us the basic information we need for the time being.

Thanks!

### CODE REVIEW

File: scrapers/USA/KY/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'KY',
+  priority: 1,
+  url: 'https://datawrapper.dwcdn.net/BbowM/23/',
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: Is this Kentucky Health Organization? If so, let's add
```suggestion
  url: 'https://datawrapper.dwcdn.net/BbowM/23/',
  source: {
    name: 'Kentucky Health Organization',
    url: 'http://wherever.this.comes.from'
  },
```

File: scrapers/USA/KY/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    const $ = await fetch.headless(this.url);
+
+    const counties = [];
+    const jsonList = [];
+
+    // Extract raw csv from link attribute on Kentucky Health Organizations data map.
+    const $rawCSVstring = decodeURIComponent($('a[class="dw-data-link"]').attr('href')).replace('data:application/octet-stream;charset=utf-8,', '');
+
+    // Convert this string into something more manageable (json)
+    const lines = $rawCSVstring.split('\n');
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: You can go ahead and import `csvParse` for this

File: scrapers/USA/KY/index.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      }
+
+      jsonList.push(obj);
+    }
+
+    // Construct counties list and return to system.
+    for (const county of jsonList) {
+      counties.push({
+        county: geography.addCounty(county.County),
+        cases: county.Total === 'null' ? 0 : parseInt(county.Total, 10),
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: use `parse.number()` -- it does a little more than a basic `parseInt`

File: scrapers/USA/KY/index.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'KY',
+  priority: 1,
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: Go ahead and add yourself as the maintainer if you want your name on the sources page and are interested in keeping this scraper up to date! Do it like this: https://github.com/lazd/coronadatascraper/blob/master/scrapers/BRA/index.js#L15-L24

File: scrapers/USA/OK/index.js [hunk 5]
```diff
@@ -14,19 +14,6 @@ const scraper = {
...
-  maintainers: [
-    {
-      name: 'Paul Boal',
-      email: 'paul.boal@amitechsolutions.com',
-      url: 'https://amitechsolutions.com',
-      github: 'paulboal',
-      country: 'USA',
-      flag: '🇺🇸'
-    }
-  ],
```
Comment (lazd) [inline scrapers/USA/OK/index.js]: Looks like an accidental merge issue?

File: scrapers/USA/MO/index.js [hunk 6]
```diff
@@ -168,14 +155,12 @@ const scraper = {
...
-          counties[countyName] = {
-            cases: casesState + casesOther
-          };
-        }
+      if (countyName in counties) {
+        counties[countyName].cases += casesState + casesOther;
+      } else {
+        counties[countyName] = {
+          cases: casesState + casesOther
+        };
```
Comment (lazd) [inline scrapers/USA/MO/index.js]: I think this is undoing changes from a recent patch.

File: scrapers/USA/KY/index.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  async scraper() {
+    const $ = await fetch.headless(this.url);
+
+    const counties = [];
+
+    // Extract raw csv from link attribute on Kentucky Health Organizations data map.
+    const csvText = decodeURIComponent($('a[class="dw-data-link"]').attr('href')).replace('data:application/octet-stream;charset=utf-8,', '');
+
+    // Parse csv.
+    const data = csvParse(csvText, { columns: true });
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: Go ahead and make this async: https://csv.js.org/parse/api/callback/

Comment (lazd) [reply scrapers/USA/KY/index.js]: Or if that's annoying, you can do it sync too :) https://csv.js.org/parse/api/sync/

**Comment by DatJord** [comment]
@lazd New changes pushed that should address the merge problems and the other requested items. Take a look and let me know if there is anything else!

**Comment by lazd** [comment]
I think this is good! Let me test locally, then will merge.

File: scrapers/USA/KY/index.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+      name: 'Jordan Holt',
+      email: 'jordholt@gmail.com',
+      github: 'DatJord',
+      country: 'USA',
+      flag: '🇺🇸'
+    }
+  ],
+  aggregate: 'county',
+  async scraper() {
+    const $ = await fetch.headless(this.url);
```
Comment (lazd) [inline scrapers/USA/KY/index.js]: Just checking, this has to be headless, yes?

**Comment by lazd** [comment]
Tested, this looks fantastic. 
image

File: scrapers/USA/KY/index.js (hunk 8 above)
Comment (lazd) [reply scrapers/USA/KY/index.js]: Yes, it seems so.

**Pull request merged**